bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -610,6 +634,17 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
```

Comment:
the kuadrant-operator does not need to create/update/delete roles and rolebindings, does it? All those resources should be part of the operator manifest bundle, thus managed by olm operator/helm